chef/cookbooks/ohai/files/default/plugins/linux/crowbar_libvirt.rb

```diff
@@ -0,0 +1,23 @@
+provides "crowbar_ohai"
+
+require_plugin "kernel"
+require_plugin "dmi"
+require_plugin "linux::s390x"
+
+libvirt_uuid = nil
+
+if kernel[:machine] == "s390x"
+  if s390x[:system][:manufacturer] == "KVM"
+    libvirt_uuid = s390x[:system][:uuid]
+  end
+else
+  manufacturer = dmi[:system][:manufacturer] rescue "unknown"
```

Comment:
While I understand the reason behind Hound not liking this construct in general. I don't really see how it is bad in this special case.

Any hint how it could be done better is really appreciated. Otherwise I'd just keep it like it is currently.